berkeleyMobileiOS/Classes/Academics/AcademicsViewController/LibraryViewController.swift

```diff
@@ -49,6 +52,8 @@ class LibraryViewController: UIViewController, GMSMapViewDelegate, CLLocationMan
         libInfo.append(getLibraryPhoneNumber())
         libInfo.append(getLibraryLoc())
         
+        types = [.none, .phone, .none]
```

Comment:
can you leave a comment here that tells future developers to update this array when they update the structure of the `libInfo` array? just so that they are linked up to some degree...